src/views/MidiView.vue

```diff
@@ -178,9 +178,24 @@ onUnmounted(() => {
               />
             </span>
           </div>
-          <div class="control checkbox-container">
-            <input type="checkbox" id="midi-velocity" v-model="midi.velocityOn" />
-            <label for="midi-velocity">Use velocity</label>
+          <div class="control checkbox-group-scoped">
+            <span>
+              <input type="checkbox" id="midi-velocity" v-model="midi.velocityOn" />
+              <label for="midi-velocity"> Use velocity </label>
+            </span>
+            <span>
+              <input type="checkbox" id="multichannel" v-model="midi.multichannelToEquave" />
+              <label for="multichannel"> Multichannel-to-equave </label>
+            </span>
+          </div>
+          <label>Settings for multichannel-to-equave mode</label>
+          <div class="control multichannel-input-scoped">                    
```

Comment:
It would be better UX if this div was collapsed or at least the inputs disabled when multichannel mode is off.

Reply:
how should I implement this, I agree but am nit sure how to do it.